### ISSUE TITLE
Possible unknown traits and rts function

### ISSUE THREAD

**Issue by DarkTl** [Xela00/PyTFall#35]
Here's the deal. Initially you cannot know which traits a girl has unless you ask her about it. Except main personality trait. About her occupation ie traits classes you already can ask directly. But you cannot ask how serious she is for example, it would be an odd question.
There are events during jobs based on traits, yeah, but MC does not see it. Only player can see those events. So it's up to dialogues only.

I have multiple lines that are based on girl traits. They are randomly selected with rts function when you talk. So if we make rts to return choosen trait (that option is needed anyway even if traits will not be hidden) and add a function that makes selected trait visible, we'll have a system of hidden traits.
Until you discover traits, they might look in the gui like "???"

**Comment by Xela00** [comment]
There is also ct function that we use to check traits. There is also a fact that MC should be able to tell a number of traits just by looking at the character.

- rts() always returns a trait, it can never return anything else.
- I'll look into setting auto-flags to traits with these functions.
- We'll have to add one or two new fields to traits for this.

I'll flag this as long term for now.

**Comment by DarkTl** [comment]
Ct functions is not an issue because I can manually add to the check a function that opens the trait. Unlike rts, where I don't know which one will be selected. The number of traits is not an issue either, they all are hidden anyway.
Rts can return occupation too btw.

**Comment by Xela00** [comment]
Yeah. I think I'll just automate that. Functions will automatically unlock traits so you will not have to do that yourself. I can install some from of simple notification as well.

This is long term task, we have more important stuff to take care of.
